app/dociql/index.js

```diff
@@ -4,6 +4,7 @@ const url = require('url')
 const fs = require("fs")
 const fetchSchema = require("./fetch-schema")
 const composePaths = require("./compose-paths")
+const composeErrorDefinitions = require("./compose-error-definitions");
```

Comment:
I would rename this to _addErrorDefinitions_

Reply:
I was following the same pattern of file names, like `composePaths`, `composeDefinitions` (which it is not used).
And actually both `addErrorDefinitions` and `composeErrorDefinitions` are not so accurate because the function is returning the entire block of definitions, not only errors

Reply:
Yes but the file ./compose-error-definitions gets the definitions by parameter and add the error definitions.
I know you have followed the current naming, just think is not clear at all and find a better way.

Reply:
done